google-cloud-security-private_ca-v1beta1/google-cloud-security-private_ca-v1beta1.gemspec

```diff
@@ -0,0 +1,37 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+require File.expand_path("lib/google/cloud/security/private_ca/v1beta1/version", __dir__)
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-security-private_ca-v1beta1"
+  gem.version       = Google::Cloud::Security::PrivateCa::V1beta1::VERSION
```

Comment:
Should this module be capitalized `PrivateCA`? (See same question on definition of `VERSION`.)

Reply:
Fixed